core/src/SYCL/Kokkos_SYCL.hpp

```diff
@@ -78,6 +79,19 @@ class SYCL {
   //! \name Functions that all Kokkos devices must implement.
   //@{
 
+  bool is_running() const {
+    const sycl::info::event_command_status sycl_status =
+        m_space_instance->m_last_event
+            .get_info<sycl::info::event::command_execution_status>();
+    switch (sycl_status) {
+      case sycl::info::event_command_status::submitted:
```

Comment:
You don't need `[[fallthrough]]`  to avoid warnings?

Reply:
No, compilers don't warn for collapsed cases.